bdb/temptable.c

```diff
@@ -79,6 +79,17 @@ struct hashobj {
     unsigned char data[/*len*/];
 };
 
+int gbl_temptable_count;
```

Comment:
object pool already keeps track of this.

Reply:
Of course, that only works when the object pool is enabled.  This count works when it is disabled.